### DISCUSSION TITLE
Is there any way to mask the credentials of kafka cluster in UI ?

### DISCUSSION THREAD

**Discussion by AayushSaini101** [provectus/kafka-ui#4498] [General]
<img width="1176" alt="Screenshot 2024-11-05 at 11 01 22 AM" src="https://github.com/user-attachments/assets/9b0195dd-cdd3-4bb8-939b-0fcc26cd6d14">
Hi team, we are running kafka ui in the docker and in future planning to host it so that other team can use it, but the credentials are clearly visible is there way to mask the credentials thanks

**Comment by Haarolean** [comment]
Hi, please be aware that this repo is not maintained (#4255). Happy to address your issues here: https://github.com/kafbat/kafka-ui

**Reply by AayushSaini101** [reply]
Thanks @Haarolean Transferred  : )

**Comment by Greatowen42** [comment]
Please I need a Google console to publish my app.
I have a budget please 

WhatsApp me
+63966 470 0031 please

**Comment by Greatowen42** [comment]
Please I need a Google console to publish my app.
I have a budget please 

WhatsApp me
+63966 470 0031 please

**Comment by Greatowen42** [comment]
Please I need a Google console to publish my app.
I have a budget please 

WhatsApp me
+63966 470 0031 please
